packages/fuel-indexer-types/src/lib.rs

```diff
@@ -29,11 +29,46 @@ pub type UInt16 = u128;
 pub type Timestamp = u64;
 pub type Charfield = String;
 pub type Boolean = bool;
-pub type Blob = Vec<u8>;
+
+#[derive(Deserialize, Serialize, Clone, Eq, PartialEq, Debug, Hash)]
```

Comment:
👌🏽 

This is fine but we'll want a `impl From<Blob> for Ve<u8>` as well (as well as a `TryFrom`)